Tests/Unit/ParseClientConfigurationTests.m

```diff
@@ -0,0 +1,92 @@
+/**
+ * Copyright (c) 2015-present, Parse, LLC.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+@import Foundation;
+
+#import "PFTestCase.h"
+#import "ParseClientConfiguration.h"
+#import "ParseClientConfiguration_Private.h"
+
+@interface ParseClientConfigurationTests : PFTestCase
+@end
+
+@implementation ParseClientConfigurationTests
+
```

Comment:
Tests for extension APIs still pending, since there is a logic difference between enabling data sharing via Parse vs ParseClientConfiguration.